moodle/Tests/fixtures/Commenting/InlineCommentAttributeAfter.php

```diff
@@ -20,4 +20,14 @@ class Example {
     public function doSomething() {
         // Do something.
     }
+
+    /**
+     * Do something with multiple attributes.
+     */
+    #[Attribute()]
+    #[Attribute()]
+    #[Attribute()]
+    public function doSomething() {
+        // Do (something else.)
+    }
```

Comment:
Location of this test is not great. This file is tells the world it is about testing attributes.

Are there tests for the existing comment punctuation logic? Should you make a separate test class?

Reply:
Agree - I was being quick and lazy
I've updated it to its own fixture, and moved the existing test to the correct location. We decided to change where we were storing fixtures so that they sit close to the test, instead of in the global fixture direcory.